source/javascript/components/s3/list.js

```diff
@@ -64,7 +64,7 @@ function S3ListController(MetadataService, Utils) {
                 values: {
                     'Enabled': 'Enabled',
                     'Disabled': 'Disabled',
-                    'Unavailable': 'Unavailable'
+                    'cinq cant poll': 'cinq cant poll'
```

Comment:
Fix punctuation on "cant"
```suggestion
                    'cinq cant poll': "cinq can't poll"
```